Meta/run.sh

```diff
@@ -213,6 +223,8 @@ if [ -z "$SERENITY_MACHINE" ]; then
         -device i82801b11-bridge,id=bridge3 -device sdhci-pci,bus=bridge3
         -device ich9-ahci,bus=bridge3
         -chardev stdio,id=stdout,mux=on
+        -qmp unix:qmp-sock,server,nowait
```

Comment:
This was removed by #11237 , consider rebasing on master or removing. Looks good otherwise, thanks for the extra comments!

Reply:
Good catch! Probably I did something funny during my rebase!
Anyway, I have to do a big refactoring/rearranging based on @ADKaster's comment.
I will include it then.